### PR TITLE
Issue 906 further clean up for exporting timetable

### PR DESCRIPTION
**Pull request by AilsaF**

### CODE REVIEW

File: app/Export/GetImages.hs [hunk 1]
```diff
@@ -28,101 +28,94 @@ getActiveGraphImage req = do
...
+convertTut (tutCode, tutSection, tutSession) =
+  (tutCode,
+   take 1 tutSection ++ "UT-" ++ drop 1 tutSection,
+   tutSession)
+
+-- | Queries the database for times regarding all lectures and tutorials,
+-- returns two lists of list of Time.
+getTimes :: ([(String, String, String)], [(String, String, String)]) -> IO ([[Time]], [[Time]])
+getTimes (selectedLecs, selectedTuts) = runSqlite databasePath $ do
   runMigration migrateAll
```
Comment (david-yz-liu) [inline app/Export/GetImages.hs]: You can remove this line.

File: app/Export/GetImages.hs [hunk 2]
```diff
@@ -28,101 +28,94 @@ getActiveGraphImage req = do
...
-  session = tutSession,
-  time = []
-  }
-
-
--- | Get data from database for selected courses
-getTimes :: ([CourseInfo], [CourseInfo]) -> IO ([CourseInfo], [CourseInfo])
-getTimes (selectedLectures, selectedTutorials) = runSqlite databasePath $ do
+convertLec :: (String, String, String) -> (String, String, String)
+convertLec (lecCode, lecSection, lecSession) =
```
Comment (david-yz-liu) [inline app/Export/GetImages.hs]: You don't need these helpers; do the conversion directly in `getLectureTime` and `getTutorialTime`.

File: app/Export/GetImages.hs [hunk 3]
```diff
@@ -28,101 +28,94 @@ getActiveGraphImage req = do
...
 
---  "STA355H1" "L0101" "F"  Time {timeField = [0.0,14.0]}
-addCourseHelper :: String -> String -> String -> [[String]] -> (Int, Int) -> [[String]]
-addCourseHelper courseCode courseSection courseSession currentSchedule (day, courseTime) =
+-- | Appends information of course to the current schedule for specified day and time.
+-- Returns new schedule.
+addCourseHelper :: (String, String, String) -> [[String]] -> (Int, Int) -> [[String]]
+addCourseHelper (courseCode, courseSection, courseSession) currentSchedule (day, courseTime) =
   let time_schedule = currentSchedule !! courseTime
       current_schedule = if (null $ time_schedule !! day) then (courseCode++courseSession++" "++courseSection) else (time_schedule !! day ++ ("&"++courseCode++courseSession++" "++courseSection))
```
Comment (david-yz-liu) [inline app/Export/GetImages.hs]: I believe we talked about storing the contents of each cell as a list of courses, rather than a formatted string.

**Comment by AilsaF** [comment]
Done implementing requested changes~

File: app/Export/ImageConversion.hs [hunk 4]
```diff
@@ -22,9 +25,9 @@ convertToImage :: String -> String -> IO
                       Maybe Handle,
                       ProcessHandle)
 convertToImage inName outName = createProcess $ CreateProcess
-                                  (ShellCommand $ "convert " ++
+                                  (ShellCommand $ "rsvg-convert " ++
```
Comment (david-yz-liu) [inline app/Export/ImageConversion.hs]: @AilsaF Please revert this change. I removed it specifically because it did not work for me.

File: app/Export/ImageConversion.hs [hunk 5]
```diff
@@ -36,19 +39,17 @@ convertToImage inName outName = createProcess $ CreateProcess
                                   False
                                   False
 
+-- removeFile name = createProcess $ CreateProcess
```
Comment (david-yz-liu) [inline app/Export/ImageConversion.hs]: You can delete this.

File: app/Export/ImageConversion.hs (hunk 4 above)
Comment (AilsaF) [inline app/Export/ImageConversion.hs]: Sorry... I always forgot to change it back 😂

**Comment by AilsaF** [comment]
Thanks a lot for the suggestion using `raw`. It works perfectly!
